app/views/dashboards/index.html.erb

```diff
@@ -89,11 +89,14 @@
                       <strong class="app-c-tag app-c-tag--success">
                         <%= t(".results.statuses.pending_payment") %>
                       </strong>
-                    <% end %>
-                    <% if transient_registration.pending_manual_conviction_check? %>
+                    <% elsif transient_registration.pending_manual_conviction_check? %>
```

Comment:
'Pending payment' and 'Pending conviction check' are not mutually exclusive conditions – sometimes a renewal can have both, and so should display both flags. However this would mean that only the payment flag would be shown.

Reply:
🤦‍♂️

Reply:
Confirmed this is back to how it was, but including the stuck status by testing in pre-prod. Ping me if you want some registration numbers